test/lit/passes/stack_switching.wast

```diff
@@ -0,0 +1,163 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;;RUN: wasm-opt -all -O3 -Oz %s -S -o - | filecheck %s
```

Comment:
Would you be able to break this up into targeted tests for the individual fixed optimization passes? Also, the new test files should be added to the list of ignored tests in scripts/fuzzing.py until scripts/fuzz_opt.py can run cleanly without that.